### PR TITLE
boards: bluepill: invert LED state

### PR DESCRIPTION
**Pull request by Teufelchen1**
tested both, macro and saul.

### CODE REVIEW

**Comment by miri64** [comment]
Thank you for your contribution, but can you please explain why this PR exists? It's nice to know that you tested it, but I don't know why.

**Comment by miri64** [comment]
(I don't want to say your PR is invalid. I'm sorry if this sounded like that. Just that it lacks some documentation)

**Comment by Teufelchen1** [comment]
Sure. I tried to turn off the PC13 LED on the bluepill with the `LED_OFF` macro. After a little bit of debugging I realised, `LED_ON` and `LED_OFF` macros are inverted.

File: boards/bluepill/include/gpio_params.h [hunk 1]
```diff
@@ -34,7 +34,8 @@ static const  saul_gpio_params_t saul_gpio_params[] =
     {
         .name = "LED",
         .pin = LED0_PIN,
-        .mode = GPIO_OUT
+        .mode = GPIO_OUT,
+        .flags = SAUL_GPIO_INVERTED
```
Comment (smlng) [inline boards/bluepill/include/gpio_params.h]: could you please add `| SAUL_GPIO_INIT_CLEAR` (see [here](https://github.com/RIOT-OS/RIOT/blob/master/boards/pba-d-01-kw2x/include/gpio_params.h#L44)) as suggested offline yesterday. This makes it explicit that the LED will be off on init when using SAUL.

Comment (smlng) [reply boards/bluepill/include/gpio_params.h]: amend changes directly

**Comment by smlng** [comment]
please squash

**Pull request merged**